### ISSUE TITLE
Try to import all issues and PRs on the old repository

### ISSUE THREAD

**Issue by jaromil** [dyne/dnscrypt-proxy#1]
Take contact with @jedisct1 if possible and import the repository that was once, or decide to start from scratch. Anyone compelled to open his/her own issues here please wait a few days for us to realise if we can port the old github repo or not.

**Comment by zeffy** [comment]
You guys might already be considering this, but it would be nice if you could get the content from the wiki too, lots of documentation is currently missing. For now it is still accessible with Google cache e.g.

**Comment by jaromil** [comment]
Today we finally noticed the archive uploaded on https://github.com/dnscrypt

It makes more sense to maintain dnscrypt there. If @jedisct1 agrees then please grant me administrator rights to the group and I'll keep working there.

**Comment by krattai** [comment]
The repo on github/dnscrypt is from the opennic community.  The Dyne and https://github.com/snorkasaurus/dnscrypt-proxy are the most up to date.

OpenNIC is still trying to decide what to do.  If this repo becomes active, than I'm sure many of us would rather leave the active project under the capable hands of Dyne.

**Comment by jaromil** [comment]
I agree coordination is needed. To virtually handshake and align commitments we can use this mailinglist https://mailinglists.dyne.org/cgi-bin/mailman/listinfo/dnscrypt

**Comment by dartraiden** [comment]
Backup of DNSCrypt wiki - https://github.com/dartraiden/dnscrypt-wiki-backup

**Comment by KrasnayaPloshchad** [comment]
Backup of former dnscrypt-proxy codebase
https://web.archive.org/web/*/https://github.com/dnscrypt-proxy/

**Comment by KrasnayaPloshchad** [comment]
Frank is prepared to receive issues and PRs now from the new repository.

**Comment by jaromil** [comment]
Frank's new work is indeed interesting. I wonder how long will take us to port plugins to it.
Meanwhile is a pity there is no answer from the opennic people.

**Comment by jedisct1** [comment]
Hi @jaromil 

I was planning to add support for the old plugin API, mainly for you. But if you think your code could be rewritten in Go, that would make things easier :)

**Comment by reelsense** [comment]
@jedisct1 

This bot is the fastest way I can think of to move GH Issues. 

https://github.com/dessant/move-issues

The one problem I see is the bot has to be authorized in the old repo.

**Comment by jaromil** [comment]
@jedisct1 many thanks of thinking of me / us :^) our fellow dev @parazyd is also proficient in go... nevertheless it would be really good if you can support C plugins.
